x/slashing/params.go

```diff
@@ -4,77 +4,114 @@ import (
 	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/params"
 )
 
-// nolint
+// Default parameter namespace
 const (
-	MaxEvidenceAgeKey           = "slashing/MaxEvidenceAge"
-	SignedBlocksWindowKey       = "slashing/SignedBlocksWindow"
-	MinSignedPerWindowKey       = "slashing/MinSignedPerWindow"
-	DoubleSignUnbondDurationKey = "slashing/DoubleSignUnbondDuration"
-	DowntimeUnbondDurationKey   = "slashing/DowntimeUnbondDuration"
-	SlashFractionDoubleSignKey  = "slashing/SlashFractionDoubleSign"
-	SlashFractionDowntimeKey    = "slashing/SlashFractionDowntime"
+	DefaultParamspace = "slashing"
 )
 
+// Parameter store key
+var (
+	KeyMaxEvidenceAge           = []byte("MaxEvidenceAge")
+	KeySignedBlocksWindow       = []byte("SignedBlocksWindow")
+	KeyMinSignedPerWindow       = []byte("MinSignedPerWindow")
+	KeyDoubleSignUnbondDuration = []byte("DoubleSignUnbondDuration")
+	KeyDowntimeUnbondDuration   = []byte("DowntimeUnbondDuration")
+	KeySlashFractionDoubleSign  = []byte("SlashFractionDoubleSign")
+	KeySlashFractionDowntime    = []byte("SlashFractionDowntime")
+)
+
+// Params - used for initializing default parameter for slashing at genesis
+type Params struct {
+	MaxEvidenceAge           time.Duration `json:"max-evidence-age"`
+	SignedBlocksWindow       int64         `json:"signed-blocks-window"`
+	MinSignedPerWindow       sdk.Dec       `json:"min-signed-per-window"`
+	DoubleSignUnbondDuration time.Duration `json:"doublesign-unbond-duration"`
+	DowntimeUnbondDuration   time.Duration `json:"downtime-unbond-duration"`
+	SlashFractionDoubleSign  sdk.Dec       `json:"slash-fraction-doublesign"`
```

Comment:
double-sign here too.. only because "doublesign" isn't actually a word (whereas "Downtime" is!)